dist/amd64/birdy/bird-wrapper.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/sh
 
-set -e
+set -ex
+
+ROUTER_ID=`ip -4 -o a | while read num intf inet addr; do
+    case $intf in
+	# Allow for "eth0", "ens5", "enp0s3" etc.; avoid "lo" and
```

Comment:
Probably needs to be configurable; what if I've got bond0, gre6, slip0, wwan3, etc...